src/terra-dev-site/guide/Component Guides/RecreatingItemCollection.guide.mdx

```diff
@@ -0,0 +1,201 @@
+import ResponsiveListandTableExample from '../../example/ResponsiveListAndTableExample?dev-site-example';
+import ResponsiveListandCellGridExample from '../../example/ResponsiveListAndCellGridExample?dev-site-example';
+import ResponsiveListandHtmlTableExample from '../../example/ResponsiveListAndHtmlTableExample?dev-site-example';
+
+# Re-creating Item Collection
+
+An Item Collection is a wrapper component designed to display data as either a table or list of items dependent on the relative container size.
+
+The `terra-item-collection` used [terra-responsive-element](/components/terra-responsive-element/responsive-element/responsive-element) to swap between it's [terra-table](/components/terra-table/table/table) and [terra-list](/components/terra-list/list/list) displays. The examples in this guide use `ActiveBreakpointContext` from [terra-application](/application/terra-application/application) to switch what is displayed based on current browser size.
+
+```javascript
+import { ActiveBreakpointContext } from 'terra-application/lib/breakpoints';
+...
+const CustomItemCollection = () => {
+  const activeBreakpoint = React.useContext(ActiveBreakpointContext);
+  ...
+  switch (activeBreakpoint) {
+    case 'tiny':
+    case 'small':
+    case 'medium':
+      return listDisplay;
+    case 'large':
+    case 'huge':
+    case 'enormous':
+    default:
+      return tableDisplay;
+  }
+};
+```
+
+This will display our list display at browser size `medium` and below and our table display on larger sizes.
+
+The examples also use [terra-clinical-item-view](/components/terra-clinical-item-view/clinical-item-view/clinical-item-view) for each cell of data which is not necessary and is used to to recreate the deprecated `terra-item-collection`.
+
+## Replicating the Deprecated Terra Clinical Item Collection displays
+
+### Creating the List version
+
+Start with a Terra `List` component and define these two props: `dividerStyles` and `role`. `dividerStyle` is used to determine if and where you want the border-color to be applied to each of the children and can have the values 'none', 'standard', 'bottom-only'. `role` is only used if the children you have in `List` are selectable, and if they are, is defined as 'listbox'.
+
+```javascript
+  const listDisplay = (
+  <List dividerStyle="none" role="listbox">
+    {children}
+  </List>);
+```
+
+Next we build the children that are going into this `<List>`. Each child should be wrapped in `terra-list`'s `<Item>` and as display in an `<ItemView>` from `terra-clinical-item-view`.
+
+```javascript
+  var listChild = (
+    <Item>
+      <ItemView
+        displays={React.Children.toArray(children)}
+      />
+    </Item>
+  );
+```
+
+## Creating Single Selectable Table version
```

Comment:
Should we start with creating the table items before implementing selectable or should the list items example above show to make the items selectable as well?

Reply:
We need to go one way or the other, it's not acceptable to have a selectable table that transforms into a non-selectable list.